### ISSUE TITLE
Provide feedback on input validation

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/vesper-to-ebird#6]

**Comment by nvcexploder** [comment]
Yes! Could we expand this issue to "Provide feedback on input validation"? Validation could include checking the settings file, ensuring the input file is there and non-empty, etc.

**Comment by RichardLitt** [comment]
Sounds good to me!

**Comment by RichardLitt** [comment]
This also applies to stations: https://github.com/RichardLitt/vesper-to-ebird/commit/15357f66671be9fef6155ee99f7517ce2c5245d7
